Backup merge

OscarTheelen opened the pull request:
Making merge to original repo for backup

OscarTheelen committed:
Merge branch 'main' into Temp_Branch_1

Commit:
Fix: wachtwoordhasher

Commit:
fix: wachtwoordhasher

Commit:
Whoops verkeerde branch
(Undo)

Commit:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus

Commit:
fix: wachtwoordhasher

Commit:
Full fix: passwordhasher

Commit:
weghalen console.log voor het fixen van de hasher

Doomayy committed:
file restructure

OscarTheelen committed:
Merge pull request #4 from DevOps-2023-Casusmaatjes/Temp_Branch_1

Temp branch 1

OscarTheelen committed:
revert restructur", still on File restructure

OscarTheelen committed:
Account registration firebase added

OscarTheelen committed:
added folder for firebase

RainbowNova committed:
Fixed the repository

- Deleted duplicate files

Doomayy committed:
grammar

Doomayy committed:
index kapoet, nu gucci

JasperSmitz committed:
Added JavaScript unit testing

- generated package.json
- Installed testing frameworks mocha and chai
- Added template test file.

JasperSmitz committed:
Update azure-container-webapp.yml

Updated to only build and deploy if the unit tests were successful.

JasperSmitz committed:
Create mocha-tests.yml

Unit test workflow
does not push to main upon failure.

JasperSmitz committed:
add workflow_dispatch to azure-container-webapp.yml

To manually run the workflow

JasperSmitz committed:
add workflow_dispatch to mocha-tests.yml

JasperSmitz committed:
Fix syntax in azure-container-webapp.yml

JasperSmitz committed:
Update mocha-tests.yml

OscarTheelen committed:
removed login popup(too much trouble) added page

OscarTheelen committed:
decorated login page

OscarTheelen committed:
installed bootstrap 5.3.0 + poppers

OscarTheelen committed:
fixed firebase on login page

OscarTheelen committed:
Tested with bootstrap on about page

OscarTheelen committed:
changed storeAccount function

Commit:
Bootstrap Grid op Profielpagina

Doomayy committed:
fixes

Doomayy committed:
wrong branch

Doomayy committed:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus

OscarTheelen committed:
added login function

OscarTheelen committed:
removed the useless firebaseconfig

OscarTheelen committed:
added dummy login

Commit:
Correcte benoeming CSS

Commit:
Correcte benoeming CSS

Commit:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Commit:
CSS hersteld

OscarTheelen committed:
removed dist from gitignore

OscarTheelen committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

thmaz committed:
Initial code game 1 added

Silencx committed:
Merge pull request #5 from DevOps-2023-Casusmaatjes/groep3-unit-testing-automation

Added unit testing automation

OscarTheelen committed:
Merge branch 'main' into groep-1-devbranch

OscarTheelen committed:
Merge pull request #6 from DevOps-2023-Casusmaatjes/groep-1-devbranch

Groep 1 devbranch

OscarTheelen committed:
Revert "Groep 1 devbranch"

OscarTheelen committed:
Merge pull request #8 from DevOps-2023-Casusmaatjes/revert-6-groep-1-devbranch

Revert "Groep 1 devbranch"

OscarTheelen committed:
test

OscarTheelen committed:
test

OscarTheelen committed:
removed test text

OscarTheelen committed:
Merge branch 'main' into groep-1-devbranch

OscarTheelen committed:
Merge attempt

OscarTheelen committed:
removed dist from gitignore

OscarTheelen committed:
added firebase folder back in

RainbowNova committed:
Merge branch 'main' into devbranch-group3

RainbowNova committed:
Removed unnecessary files

RainbowNova committed:
Fixed references after previously removing files

RainbowNova committed:
Replaced & Fixed the header-component in index.html

- index.html still had the old code instead of using the component. Replaced it with the component + fixed the buttons not working.

RainbowNova committed:
Fixed a minor mistake in filename reference

RainbowNova committed:
Able to create divs through button presses

RainbowNova committed:
Fixed added divs not being added correctly

- Divs now get added to the correct div-container.

Silencx committed:
Updated Dev brench to reflect Main brench

OscarTheelen committed:
Groep 1 devbranch Merge (#10)

* Added Bootstrap
* Removed dist from gitignore
* Added firebase folder back in

Silencx committed:
Updated branch to reflex Main

Silencx committed:
Merge pull request #11 from DevOps-2023-Casusmaatjes/groep2-devbranch

Merging from Devbrench Groep 2 in Main

Silencx committed:
Update nicer_header.js

Silencx committed:
Update index.html

Silencx committed:
Update StorePage.html

OscarTheelen committed:
Merge branch 'main' into groep-1-devbranch

OscarTheelen committed:
Merge branch 'main' into groep-1-devbranch

RainbowNova committed:
Last attempts with php

- Didn't work. PhP won't run. JS Module Files deprecated bug.

Roelkuhh043 committed:
bootstrap installed

bootstrap erin gepleurd

thmaz committed:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep4-devbranch

OscarTheelen committed:
login button switch

OscarTheelen committed:
header added to index

MasterDisaster7 committed:
Code Cleanup

Roelkuhh043 committed:
update naar bootstrap

update css naar bootstrap

Silencx committed:
Merge pull request #12 from DevOps-2023-Casusmaatjes/groep2-DevBrench

Groep2 dev brench merge to main

Silencx committed:
Create ReviewRequest.yml

Silencx committed:
Added src file for Review bot

Silencx committed:
Delete ReviewRequest.yml

OscarTheelen committed:
Merge branch 'main' into main

JasperSmitz reviewed (approved):
👍

RainbowNova committed:
Merge branch 'devbranch-group3'

RainbowNova committed:
Removed unneeded files

- Also started testing Bootstrap and following tutorials

OscarTheelen committed:
Update index.html

RainbowNova committed:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus

RainbowNova committed:
Created main forum page

- Using Bootstrap
- Using JS components
- Not using common.css because of current Bootstrap incompatibility issues.

RainbowNova committed:
Fixed container for forum pages

Container had some problems regarding  the temporary submenu and rows/cols.

Roelkuhh043 committed:
navbar verbetert

navbar verbetert met bootsrtap

Doomayy reviewed (approved):
👍

MasterDisaster7 reviewed (approved):
Lekker bezig jonges

OscarTheelen committed:
edited css login

OscarTheelen committed:
edited css login

Commit:
Profilel layout test

Commit:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Commit:
Layout profielpagina verbeterd

RainbowNova committed:
Made Add Category button

RainbowNova reviewed (approved):
LGTM 👍 
Suggestion to automate this to make it automatically backup daily?

Commit:
container / table
omdraaien om best practice te volgen

Doomayy committed:
mysql integration test

Doomayy committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Commit:
layout profiel headers

Commit:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Commit:
Layout profiel headers 2.0

Commit:
Bepaalde onderdelen van profielpagina verwijderd

thmaz committed:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep4-devbranch

thmaz committed:
added localstorage for pointsand level

thmaz committed:
localstorage points

JGeraats2002 committed:
FlappyBird Code werkend in html

JGeraats2002 committed:
Something went wrong with semicolon; is fixed

Commit:
Layout vrienden voorbereid op content

Doomayy committed:
changes

Doomayy committed:
changes

Doomayy committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

MasterDisaster7 committed:
cleaning

MasterDisaster7 committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

MasterDisaster7 committed:
basic sql code

Commit:
Layout aangepast profielpagina (overzichtelijker)

Commit:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Commit:
Profielpagian geupdate + snake als game voorbereid

Commit:
Voorbereidingen op pull request

Commit:
Scrollable container voor vriendenlijst

OscarTheelen committed:
Create manual_deploy.yml

this yml file can be used to manually deploy from a branch other than main for testing purposes

OscarTheelen committed:
Update manual_deploy.yml

Roelkuhh043 committed:
Store page fix + Game page made

bootstrap gefix bij de store page en een basis van de game page gemaakt

OscarTheelen committed:
Merge pull request #15 from DevOps-2023-Casusmaatjes/Groep-5

Groep 5

RainbowNova committed:
Attempted to connect to the DB

- Python can connect to the DB
- Has password hardcoded atm (huge security risk)
- Can't connect the HTML to the database however

RainbowNova committed:
Added the missing node_module files

JasperSmitz committed:
Add posts frontend and comment system

No connection to database, none of the comments 'posted' are saved as of yet.

thmaz committed:
update files

thmaz committed:
update + fixes

thmaz committed:
Merge branch 'main' into groep4-devbranch

Blobert123 committed:
Merge pull request #20 from DevOps-2023-Casusmaatjes/groep2-DevBrench

Groep2 dev brench

thmaz committed:
aanpassingen teruggedraaid

thmaz committed:
Merge pull request #19 from DevOps-2023-Casusmaatjes/Ozziehman-patch-1

Create manual_deploy.yml

ProgrammerGhostPrK committed:
Merge pull request #18 from DevOps-2023-Casusmaatjes/groep-1-devbranchh

groep-1-devbranch

OscarTheelen committed:
Merge branch 'main' into group3-conversion-to-bootstrap

OscarTheelen committed:
Merge pull request #22 from DevOps-2023-Casusmaatjes/group3-conversion-to-bootstrap

Group3 conversion to bootstrap, attempted connecting to the MySQL DB & added missing files

OscarTheelen committed:
working docker-compose

OscarTheelen committed:
Merge branch 'main' into groep-1-devbranch

OscarTheelen committed:
profile change

OscarTheelen committed:
added friend adding and changed style a bit

OscarTheelen committed:
styles fixed

OscarTheelen committed:
transparent scrollbar

OscarTheelen committed:
Merge branch 'main' into groep4-devbranch

OscarTheelen committed:
Update manual_deploy.yml

OscarTheelen committed:
Merge pull request #23 from DevOps-2023-Casusmaatjes/Ozziehman-patch-2

Update manual_deploy.yml

OscarTheelen committed:
Merge remote-tracking branch 'origin/main' into groep-1-devbranch

OscarTheelen committed:
Update manual_deploy.yml

OscarTheelen committed:
Merge pull request #24 from DevOps-2023-Casusmaatjes/Ozziehman-patch-1

Update manual_deploy.yml

OscarTheelen committed:
Merge remote-tracking branch 'origin/main' into groep-1-devbranch

OscarTheelen committed:
storepage margin added

thmaz committed:
Bugfixes, updated sprites

JGeraats2002 committed:
Changed directory of flappy bird code

OscarTheelen committed:
bootstrap integrated into navbar

OscarTheelen committed:
fixed animation bug

thmaz committed:
Updates to layout and style

OscarTheelen committed:
fixed header having color

thmaz committed:
new header added

OscarTheelen committed:
replaced bootstrap 4.0.0 with 5.3.0

thmaz committed:
changes

thmaz committed:
changes

OscarTheelen committed:
containerized header

thmaz committed:
general fixes

thmaz committed:
Merge branch 'main' into groep4-devbranch

thmaz committed:
Merge remote-tracking branch 'origin/main' into groep4-devbranch

thmaz committed:
ongebruikte bestanden verwijderd

JGeraats2002 committed:
test

JGeraats2002 committed:
Directory for flappybird game

OscarTheelen committed:
docker-compose fixed

OscarTheelen committed:
dockerfilename fix

OscarTheelen committed:
fixed syntax

OscarTheelen committed:
re-added node modules

OscarTheelen committed:
firebase.js removed

MasterDisaster7 committed:
docker port added

MasterDisaster7 committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

JGeraats2002 committed:
All variables made window-dependent

MasterDisaster7 committed:
firebase recode

OscarTheelen committed:
Merge pull request #28 from DevOps-2023-Casusmaatjes/groep-1-devbranch

Groep 1 devbranch

OscarTheelen committed:
Merge branch 'main' into groep4-devbranch

MasterDisaster7 committed:
Added backend for registration with API connection

JasperSmitz committed:
Merge pull request #29 from DevOps-2023-Casusmaatjes/groep-1-devbranch

Added backend for registration with API connection

JasperSmitz committed:
Merge pull request #27 from DevOps-2023-Casusmaatjes/Groep-5

Groep 5

Raffelke committed:
Merge pull request #26 from DevOps-2023-Casusmaatjes/groep4-devbranch

Groep4 devbranch

Commit:
Clickable image for game

OscarTheelen committed:
conributors added

OscarTheelen committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

OscarTheelen committed:
homepage slightly edited

Doomayy committed:
terms of service

Doomayy committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

Doomayy committed:
agree to tos required

JGeraats2002 committed:
Feedback page

OscarTheelen committed:
base login

OscarTheelen committed:
working login with changing logout/login buttons

OscarTheelen committed:
edited permissions

OscarTheelen committed:
error fixed in login

OscarTheelen committed:
edited profile page to partially load in user

thmaz committed:
Merge branch 'groep4-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep4-devbranch

thmaz committed:
Merge branch 'main' into groep4-devbranch

thmaz committed:
Fixed screen flicker, updated snake graphics, controls WASD

thmaz committed:
sound effect

thmaz committed:
updated sounds

Silencx committed:
Updated JS Code

Added JS code that allows the game and store page to be dynamically updated based on the information that the db contains

Silencx committed:
Merge pull request #30 from DevOps-2023-Casusmaatjes/Groep-5

Feedback page

Silencx committed:
Merge pull request #31 from DevOps-2023-Casusmaatjes/Development-Branch-Groep-2

Updated JS Code

Silencx committed:
Fixed Css change

Fixed a problem with a ccs file name that got changed

OscarTheelen committed:
Merge pull request #32 from DevOps-2023-Casusmaatjes/Development-Branch-Groep-2

Fixed Css change

Silencx committed:
Shopping Cart Page

Added robert functionality into our dev brench

Commit:
2de gane op profiel

thmaz committed:
mute button

RainbowNova committed:
failed attempt at loading posts

OscarTheelen committed:
register reset

OscarTheelen committed:
register reset

OscarTheelen committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

JGeraats2002 committed:
Added name for page

MasterDisaster7 committed:
Reworked and fixed the login

MasterDisaster7 committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

OscarTheelen committed:
margin ajusted

thmaz committed:
button update

JGeraats2002 committed:
Smoother to play and faster pipes. Difficulty is nicer to play

RainbowNova committed:
failed to make DB work

OscarTheelen committed:
Unused project files deleted

OscarTheelen committed:
Merge branch 'groep-1-devbranch' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep-1-devbranch

YvoSchutgens committed:
attemt post page

Attempting creating post page

YvoSchutgens committed:
Merge branch 'groep3-devbranch-new' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep3-devbranch-new

OscarTheelen committed:
changed pf page for testing

OscarTheelen committed:
working login ( still in frontend )

OscarTheelen committed:
Merge pull request #33 from DevOps-2023-Casusmaatjes/Groep-5

Groep 5

YvoSchutgens committed:
Changing post creation page

Attempt at changing post creation page

thmaz committed:
opmaak code verbeterd

JasperSmitz committed:
Clean up files

removed unused files

JasperSmitz committed:
Renamed to CreatePost.html

MasterDisaster7 committed:
Merge pull request #34 from DevOps-2023-Casusmaatjes/groep-1-devbranch

Addition of login, registration and user session

thmaz committed:
Merge branch 'main' of https://github.com/DevOps-2023-Casusmaatjes/Jaar2Blok4Casus into groep4-devbranch

thmaz committed:
removed unused code

thmaz committed:
removed unused code

thmaz committed:
removed unused files

JasperSmitz committed:
Add category loading

i think

thmaz committed:
Merge pull request #35 from DevOps-2023-Casusmaatjes/groep4-devbranch

Groep4 devbranch

JasperSmitz committed:
Fixed categories?

Silencx committed:
Added Feedback Page

Added functionality for a feedback page

JasperSmitz committed:
Testing backend

using storepage.js code to test forum backend dont worry too much about it

JasperSmitz committed:
Merge pull request #36 from DevOps-2023-Casusmaatjes/Development-Branch-Groep-2

Feedback / Storepage / ShoppingCart

JasperSmitz committed:
Update Forum.html

JasperSmitz committed:
Changed Window Title

JasperSmitz committed:
Merge branch 'main' into groep3-devbranch-new

JasperSmitz committed:
Fixed Categories

automatically gathers categories from backend and displays them on screen

JasperSmitz committed:
Update CategoryPage.js

Silencx committed:
Added CreateGamePage

JasperSmitz committed:
Update CategoryPage.html

JasperSmitz committed:
Merge pull request #37 from DevOps-2023-Casusmaatjes/groep3-devbranch-new

Groep3 devbranch new

JasperSmitz committed:
Merge pull request #38 from DevOps-2023-Casusmaatjes/Development-Branch-Groep-2

Added CreateGamePage
